django/utils/decorators.py

```diff
@@ -30,42 +32,56 @@ def _multi_decorate(decorators, method):
     a single decorator or an iterable of decorators.
     """
     if hasattr(decorators, "__iter__"):
-        # Apply a list/tuple of decorators if 'decorators' is one. Decorator
-        # functions are applied so that the call order is the same as the
-        # order in which they appear in the iterable.
+        # If 'decorators' is a list or tuple, reverse it to apply in correct order.
         decorators = decorators[::-1]
     else:
+        # If it's a single decorator, make it a list.
```

Comment:
I think this comment just says what the code below it does no? (I don't think it adds anything)